src/app/views/apps/modals/app-settings-modal.html

```diff
@@ -26,13 +26,23 @@
           </div>
         </div>
       </div>
-
       <!-- Add on settings -->
       <div ng-show="helper.isAddOnSettingShown(app)">
         <hr>
         <button ng-click="helper.addOnSettingLauch(app)" class="btn btn-warning">{{ 'mno_enterprise.templates.impac.dock.settings.access_add-on_settings' | translate }}</button>
       </div>
-
+      <!-- Billing Details -->
+      <div ng-show="helper.canDisplayBilling()">
+        <hr>
+        <div ng-show="!!app.free_trial_end_at">
+            {{ 'mno_enterprise.templates.impac.dock.settings.free_trial' | translate }}: {{ app.free_trial_end_at }}
+        </div>
+        <div ng-show="!app.free_trial_end_at && app.per_user_licence">
+          <ng-pluralize count="app.licences_count" when="{'0': '{{ &quot;mno_enterprise.templates.impac.dock.settings.no_users&quot; | translate }}', '1': '{{ &quot;mno_enterprise.templates.impac.dock.settings.1_user&quot; | translate }}', 'other': '{{app.licences_count | number}} {{ &quot;mno_enterprise.templates.impac.dock.settings.users&quot; | translate }}'}"
```

Comment:
Heys @alexnoox , this is the best solution I found to solve the problem of the i18n within the ng-pluralize. I have not checked with @clemthenem  as I thought this could be enough. Please confirm this is ok, otherwise I will look for another solution or ask clement straight away.

Thanks!